addons/web/controllers/main.py

```diff
@@ -1522,6 +1522,8 @@ def from_data(self, fields, rows):
                 cell_style = base_style
                 if isinstance(cell_value, basestring):
                     cell_value = re.sub("\r", " ", cell_value)
+					# Excel supports a maximum of 32767 characters in each cell:
```

Comment:
You have an indentation problem here, this is because you are using tabs while we are using spaces in the files